### DISCUSSION TITLE
Is there a way to disable the docs frontend?

### DISCUSSION THREAD

**Discussion by NightMachinery** [fastapi/fastapi#7389] [Questions]
I don't want the docs, they just consume extra resources. How can I turn them off?

**Comment by phy25** [comment]
Please follow thr issue template and there should be an answer.

**Comment by NightMachinery** [comment]
After searching more thoroughly through the repository, I have found an answer; Setting `export OPENAPI_URL=` in the environment of the running app. There is more info [here](https://fastapi.tiangolo.com/advanced/conditional-openapi/).

@phy25 Sorry, the search wasn't (and isn't) working for me on https://fastapi.tiangolo.com, and I don't know why I forgot to search Github. I usually do that. Thanks!

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :clap: :bow: 

Thanks for reporting back and closing the issue @NightMachinary  :+1:
